backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -562,16 +599,20 @@ public Project SetStatus(Project project, Workflow workflow, ProjectStatus statu
             switch (toStatus.Code)
             {
                 case ("SU"): // Submit
+                    this.User.ThrowIfNotAuthorized(Permissions.DisposeRequest, "User does not have permission to submit project.");
```

Comment:
Only users with `dispose-request` claim can submit.  This means the APP needs to be updated to disable the **Submit** button if the user does not have permission.